slsReceiverSoftware/include/sls/Receiver.h

```diff
@@ -51,8 +51,7 @@ class Receiver : private virtual slsDetectorDefs {
      * users get data to write depending on call backs registered
      */
     void registerCallBackStartAcquisition(int (*func)(std::string, std::string,
```

Comment:
Would it work with a const std::string&? This could be more efficient depending on what the users want to do with the names.

Reply:
It would work, but I am not aware if the users are making copies to manipulate the strings.

Reply:
I can change to this though, as the norm would be to not manipulate the strings I assume. The user gets to choose fpath, fname and findex.

Reply:
done